arbnode/espresso/submitter/original_submitter.go

```diff
@@ -0,0 +1,779 @@
+package submitter
```

Comment:
nit: can we change the name from `original_submitter` not sure what original means here?

Reply:
I named this as `original_submitter.go`, specifically as it contains the data structure named with `OriginalSubmitter`.

For the underlying reason of the name, please see my response on the comment of the struct's name itself.